### PR TITLE
[web] merge the README files into one

### PR DESCRIPTION
**Pull request by yjbanov**
It's easier to search for information when it's in one place. This PR also updates the README by fixing wording and typos, and removing references to options that no longer exist (e.g. `--unit-tests_only`, `--integration-tests-only`, options for browser versions, etc).

### CODE REVIEW

File: lib/web_ui/README.md [hunk 1]
```diff
@@ -27,3 +177,58 @@ directly), follow these steps to roll to the new version:
...
+In that case, any invocation to `felt` will fail with:
+
+```
+Can't load Kernel binary: Invalid kernel binary format version.
+```
+
+The solution is to delete the cached `felt.snapshot` files under `lib/web_ui`:
+
+```
+rm .dart_tool/felt.snapshot*
```
Comment (ditman) [inline lib/web_ui/README.md:213]: I now use: 

```
FELT_USE_SNAPSHOT=false felt ...
```

That usually fixes the tool from that moment on.

**Review by ditman — APPROVED** [review]
We had a chat the other day about some of the ways we work with `felt`, should we use this opportunity to bring the doc up to speed, or is it sufficiently updated?

**Review by mdebbar — APPROVED** [review]

File: lib/web_ui/README.md (hunk 1 above)
Comment (yjbanov) [reply lib/web_ui/README.md:213]: That fixes the kernel mismatch issue but it disables the snapshot, which slows down felt start-up time:

```
$ time felt test test/alarm_clock_test.dart 
real	0m10.601s
user	0m12.223s
sys	0m2.359s

$ FELT_USE_SNAPSHOT=false time felt test test/alarm_clock_test.dart
real	0m21.578s
user	0m27.702s
sys	0m4.177s
```

So I wouldn't recommend it.

**Comment by yjbanov** [comment]
LUCI status is false due to https://github.com/flutter/flutter/issues/90569. Merging.

**Pull request merged**